docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.6"
+
+services:
+  marklogic:
+    image: marklogic.azurecr.io/marklogic:latest
+    volumes:
+    - type: volume
+      source: marklogic_data
+      target: /data
+    ports:
+    - 7997-8020:7997-8020/tcp
+    - 8855-8856:8855-8856/tcp
+  gradle:
+    image: gradle:alpine
+    depends_on:
+      - marklogic
+    volumes:
+    - type: bind
+      source: .
+      target: /home/gradle/project
+    working_dir: /home/gradle/project
+    command: "gradle -PmlHost=marklogic mlInit mlInstallAdmin mlDeploy"
+  mysql:
+    image: mysql:8
+    command: --default-authentication-plugin=mysql_native_password
+    environment:
+    - MYSQL_ROOT_PASSWORD=password
+    volumes:
+    - type: bind
+      source: ./data/sakila-db
+      target: /docker-entrypoint-initdb.d
+    ports:
+    - 3306:3306
+  phpmyadmin:
```

Comment:
This will start up a myPhpAdmin container that gives you MySQL Workbench functionality in a web user interface.  Just hit port 8080 on your host machine.  Credentials are root/password